### PR TITLE
[ES] add content/es/auto_scaling.md

### PR DESCRIPTION
**Pull request by gamba47**
PR for issue #565

Done.

### CODE REVIEW

**Comment by jihoon-seo** [comment]
@gamba47 Thank you for your contribution!

One should be able to see the preview of your work at https://deploy-preview-705--cncfglossary.netlify.app/es/auto_scaling/ ,
but I checked that I cannot.

It seems that this is because you started your work from `main` branch, not `dev-es` branch.

Referring [this comment](https://github.com/cncf/glossary/pull/688#issuecomment-1088360208):
> You can close this PR and fetch the upstream `dev-es` branch on your forked repo again, so you're aligned. Then start from scratch: open a new branch from the `dev-es` branch, add the new term, and raise another PR which aims the `dev-es` branch as a merge target.

File: content/es/auto_scaling.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Autoscaling
+status: Completed
+category: property
+---
+
+El Autoescalado es la habilidad de un sistema para [escalar](/scalability) automaticamente, en terminos de recursos computacionales. Con un sistema de autoescalado, los recursos son agregados automáticamente cuando se necesitan y pueden escalar para cumplir con la demanda fluctuante de los usuarios. El proceso de autoescalado varía y es configurable para escalar basado en diferentes métricas, como son la memoria or el uso de CPU. Los servicios gestionados en la nube son los que están asociados típicamente con esta funcionalidad de autoescalado pero tambien hay opciones e implementaciones disponibles para la mayoría de los servicios existentes.
+
+Previamente, la infraestructura y las aplicaciones eran diseñadas para considerar los picos de uso del sistema. Esta arquitectura implicaba que había más recursos que eran subutilizados o con cambios inelasticos frente a la demanda de los usuarios. La inelasticidad en este caso, significa más costos para el negocio y pérdida de negocios debido a problemas relacionados a la sobredemanda.
```
Comment (raelga) [inline content/es/auto_scaling.md:9]:
```suggestion
Previamente, la infraestructura y las aplicaciones eran diseñadas para considerar los picos de uso del sistema. Esta arquitectura implicaba que había más recursos que eran infrautilizados o con cambios inelásticos frente a la demanda de los usuarios. La inelasticidad en este caso, incrementa el coste y puede suponer una pérdida de negocios debido a problemas de capacidad.
```

File: content/es/auto_scaling.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+Aprovechando la nube, [virtualizando](/virtualization), y [contenerizando](/containerization/) aplicaciones y sus dependencias, las organizaciones pueden construir aplicaciones que escalan de manera acorde a la demanda de los usuarios. Se pueden monitorear la demanda de las aplicaciones y de manera automática escalar las mismas, proporcionando una experiencia al usuario final óptima. Tomei de ejemplo el aumento de la audiencia de Netflix todos los dias viernes por la tarde. Escalar significa agregar más recursos dinámicamente: por ejemplo, incrementando el numero de servidores permitiendo que más usuarios visualizando videos y desescalando una vez que el consumo se ha normalizado.
```
Comment (raelga) [inline content/es/auto_scaling.md:11]:
```suggestion
Aprovechando la nube, [virtualizando](/virtualization) y [contenerizando](/containerization/) aplicaciones y sus dependencias, las organizaciones pueden construir aplicaciones que escalan de manera acorde a la demanda de los usuarios. Se pueden monitorear la demanda de las aplicaciones y de manera automática escalar las mismas, proporcionando una experiencia al usuario final óptima. Tome de ejemplo el aumento de la audiencia de Netflix todos los viernes por la tarde. Escalar significa agregar más recursos dinámicamente: por ejemplo, incrementando el número de servidores permitiendo que más usuarios visualizando videos y reduciéndolos una vez que el consumo se ha normalizado.
```

**Review by raelga — CHANGES_REQUESTED** [review]
La review añade las cabeceras a cada sección, corrige algunos errores tipográficos y cambia algunas frases para que sean más claras.

**Comment by gamba47** [comment]
Hi @jihoon-seo !   How can i change to the other branch?

I made 3 translations i will do more with this concept. Thanks for your time

**Review by raelga — APPROVED** [review]

**Comment by jihoon-seo** [comment]
@gamba47 
This is how to create another PR:
1. On your development environment, pull all branches from upstream repo:
  `git pull`
2. Switch to `dev-es` branch:
  `git checkout dev-es`
3. Create a new feature branch (that will contain your new commit) from `dev-es` branch:
  `git checkout -b <new-branch-name>`
4. Add a new file (`content/es/auto_scaling.md`), and commit:
  `git add content/es/auto_scaling.md` (Add the version that @raelga's comments are reflected. 😊)
  `git commit -m "<Some commit message>"`
5. Push your new feature branch to your forked GitHub repo (e.g. origin):
  `git push origin <new-branch-name>`
6. On GitHub web UI, open a new PR, while setting the merge target branch as `dev-es`. 😊
7. After that, this PR gets okay to close. 😊

**Comment by electrocucaracha** [comment]
> @gamba47 This is how to create another PR:
> 
>     1. On your development environment, pull all branches from upstream repo:
>        `git pull`
> 
>     2. Switch to `dev-es` branch:
>        `git checkout dev-es`
> 
>     3. Create a new feature branch (that will contain your new commit) from `dev-es` branch:
>        `git checkout -b <new-branch-name>`
> 

What about to cherry-pick that change?

```bash
git pull
git checkout -b dev-es origin/dev-es
git branch -m auto_scalling draft
git branch auto_scalling
git checkout auto_scalling
git cherry-pick 7edd30c837d0f475e44395ff36e6b4c6b9ae0742
git push origin auto_scalling -f
```

>     4. Add a new file (`content/es/auto_scaling.md`), and commit:
>        `git add content/es/auto_scaling.md` (Make sure that you add the version that @raelga's comments are reflected. 😊)
>        `git commit -m "<Some commit message>"`
> 
>     5. Push your new feature branch to your forked GitHub repo (e.g. origin):
>        `git push origin <new-branch-name>`
> 
>     6. On GitHub web UI, open a new PR, while setting the merge target branch as `dev-es`. 😊
> 
>     7. After that, this PR gets okay to close. 😊

**Comment by gamba47** [comment]
Let me know if you need a change in mi pull requests or if i do this on new pr only

**Comment by jihoon-seo** [comment]
Instead of open a new PR, updating this PR looks also good to me. 😊

Meanwhile, following @electrocucaracha's instruction, I got an error:
```
❯ git checkout -b dev-es origin/dev-es
fatal: 'origin/dev-es' is not a commit and a branch 'dev-es' cannot be created from it
```

For me, this worked (@electrocucaracha Please check whether below commands look good to you!):
```
git pull
git checkout dev-es
git branch -m auto_scalling draft
git branch auto_scalling
git checkout auto_scalling
git cherry-pick 7edd30c837d0f475e44395ff36e6b4c6b9ae0742
git push origin auto_scalling -f
```

**Comment by gamba47** [comment]
Updating this means change where i merge my changes? sorry i'm a newbie in this git field.

**Comment by jihoon-seo** [comment]
@gamba47 
> Updating this means change where i merge my changes?

Yes, 'Updating this' means that you change the merge target branch of this PR to `dev-es`.

To do so, follow this:

1. On your development environment,

```
git pull
git checkout dev-es
git branch -m auto_scalling draft
git branch auto_scalling
git checkout auto_scalling
git cherry-pick 7edd30c837d0f475e44395ff36e6b4c6b9ae0742
git push origin auto_scalling -f
```

2. On this page, change the merge target branch of this PR to `dev-es`:
image
image
image

**Review by raelga — APPROVED** [review]

**Comment by jihoon-seo** [comment]
In this case, @raelga's changing the merge target branch to `main` only seems okay without problem.
(cf. #696)

Although the preview is not available on https://deploy-preview-705--cncfglossary.netlify.app/es/auto_scaling/ 
(which is because the Netlify rebuild is triggered by someone's pushing new commits to the branch),
I guess that this page will be displayed well after all.

@gamba47 @electrocucaracha @raelga Thank you for your contributions!

File: content/es/auto_scaling.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+title: Autoscaling
+status: Completed
+category: property
+---
+
+El Autoescalado es la habilidad de un sistema para [escalar](/scalability) automaticamente, en terminos de recursos computacionales. Con un sistema de autoescalado, los recursos son agregados automáticamente cuando se necesitan y pueden escalar para cumplir con la demanda fluctuante de los usuarios. El proceso de autoescalado varía y es configurable para escalar basado en diferentes métricas, como son la memoria or el uso de CPU. Los servicios gestionados en la nube son los que están asociados típicamente con esta funcionalidad de autoescalado pero tambien hay opciones e implementaciones disponibles para la mayoría de los servicios existentes.
```
Comment (krol3) [inline content/es/auto_scaling.md:7]:
```suggestion
El auto escalado es la habilidad de un sistema para [escalar](/scalability) automaticamente, en términos de recursos computacionales. Con un sistema de auto escalado, los recursos son agregados automáticamente cuando se necesitan y pueden escalar para cumplir con la demanda fluctuante de los usuarios. El proceso de auto escalado varía y es configurable para escalar basado en diferentes métricas, como son la memoria o el uso de CPU. Los servicios gestionados en la nube son los que están asociados típicamente con esta funcionalidad de escalado automático pero también hay opciones e implementaciones disponibles para la mayoría de los servicios existentes.
```

File: content/es/auto_scaling.md (hunk 1 above)
Comment (krol3) [inline content/es/auto_scaling.md:9]:
```suggestion
Previamente, la infraestructura y las aplicaciones eran diseñadas para considerar los picos de uso del sistema. Esta arquitectura implicaba que había más recursos que eran subutilizados o con cambios inelásticos frente a la demanda de los usuarios. La inelasticidad en este caso significa más costos para el negocio y pérdida de negocios debido a problemas relacionados a la sobredemanda.
```

File: content/es/auto_scaling.md (hunk 2 above)
Comment (krol3) [inline content/es/auto_scaling.md:11]:
```suggestion
Aprovechando la nube, [virtualizando](/virtualization), y [contenerizando](/containerization/) aplicaciones y sus dependencias, las organizaciones pueden construir aplicaciones que escalan de manera acorde a la demanda de los usuarios. Se pueden monitorear la demanda de las aplicaciones y de manera automática escalar las mismas, proporcionando una experiencia al usuario final óptima. Tomemos el ejemplo del aumento de la audiencia de Netflix todos los viernes por la noche. El escalado automático significa agregar más recursos dinámicamente: por ejemplo, incrementando el número de servidores que permitirán una mayor transmisión de los videos y reduciendo los servidores una vez que el consumo se ha normalizado.
```

**Review by krol3 — COMMENTED** [review]
Coloque algunas sugerencias, encontre el termino auto escalado o escalado automaticamente más usado que autoescalado.

File: content/es/auto_scaling.md (hunk 3 above)
Comment (gamba47) [reply content/es/auto_scaling.md:7]: Hola @krol3 gracias por la review, no pude encontrar una palabra que ayude a entender bien el concepto de auto scaling, la traduccion literal de Auto Escalado para mi tiene sabor a poco, porque leyendo varias veces las lineas pareciera que solo se escala "agregando recursos" y no hay info acerca de que puede disminuir la cantidad de recursos. Se te ocurre alguna idea? vamos por tu cambio, pero me parece que se podria mejorar un poco para no perder el espirituo de la idea.

**Review by gamba47 — COMMENTED** [review]
Done

**Review by gamba47 — COMMENTED** [review]
Done

File: content/es/auto_scaling.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+title: Autoscaling
+status: Completed
+category: property
+---
+
+El autoescalado es la habilidad de un sistema para [escalar](/scalability) automáticamente, en términos de recursos computacionales. Con un sistema de autoescalado, los recursos son agregados automáticamente cuando se necesitan y pueden escalar para cumplir con la demanda fluctuante de los usuarios. El proceso de autoescalado varía y es configurable para escalar basado en diferentes métricas, como son la memoria o el uso de CPU. Los servicios gestionados en la nube son los que están asociados típicamente con esta funcionalidad de autoescalado pero también hay opciones e implementaciones disponibles para la mayoría de los servicios existentes.
```
Comment (raelga) [inline content/es/auto_scaling.md:7]:
```suggestion
## ¿Qué es esto?
El autoescalado es la habilidad de un sistema para [escalar](/scalability) automáticamente, en términos de recursos computacionales. Con un sistema de autoescalado, los recursos son agregados automáticamente cuando se necesitan y pueden escalar para cumplir con la demanda fluctuante de los usuarios. El proceso de autoescalado varía y es configurable para escalar basado en diferentes métricas, como son la memoria o el uso de CPU. Los servicios gestionados en la nube son los que están asociados típicamente con esta funcionalidad de autoescalado pero también hay opciones e implementaciones disponibles para la mayoría de los servicios existentes.
```

File: content/es/auto_scaling.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+Previamente, la infraestructura y las aplicaciones eran diseñadas para considerar los picos de uso del sistema. Esta arquitectura implicaba que había más recursos que eran infrautilizados o con cambios inelásticos frente a la demanda de los usuarios. La inelasticidad en este caso, incrementa el coste y puede suponer una pérdida de negocios debido a problemas de capacidad.
```
Comment (raelga) [inline content/es/auto_scaling.md:9]:
```suggestion
## El problema que aborda
Previamente, la infraestructura y las aplicaciones eran diseñadas para considerar los picos de uso del sistema. Esta arquitectura implicaba que había más recursos que eran infrautilizados o con cambios inelásticos frente a la demanda de los usuarios. La inelasticidad en este caso, incrementa el coste y puede suponer una pérdida de negocios debido a problemas de capacidad.
```

File: content/es/auto_scaling.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+Aprovechando la nube, [virtualizando](/virtualization) y [contenerizando](/containerization/) aplicaciones y sus dependencias, las organizaciones pueden construir aplicaciones que escalan de manera acorde a la demanda de los usuarios. Se pueden monitorear la demanda de las aplicaciones y de manera automática escalar las mismas, proporcionando una experiencia al usuario final óptima. Tome de ejemplo el aumento de la audiencia de Netflix todos los viernes por la tarde. Escalar significa agregar más recursos dinámicamente: por ejemplo, incrementando el número de servidores permitiendo que más usuarios visualizando videos y reduciéndolos una vez que el consumo se ha normalizado.
```
Comment (raelga) [inline content/es/auto_scaling.md:11]:
```suggestion 
## ¿Cómo Ayuda?
Aprovechando la nube, [virtualizando](/virtualization) y [contenerizando](/containerization/) aplicaciones y sus dependencias, las organizaciones pueden construir aplicaciones que escalan de manera acorde a la demanda de los usuarios. Se pueden monitorear la demanda de las aplicaciones y de manera automática escalar las mismas, proporcionando una experiencia al usuario final óptima. Tome de ejemplo el aumento de la audiencia de Netflix todos los viernes por la tarde. Escalar significa agregar más recursos dinámicamente: por ejemplo, incrementando el número de servidores permitiendo que más usuarios visualizando videos y reduciéndolos una vez que el consumo se ha normalizado.
```

**Review by raelga — CHANGES_REQUESTED** [review]

**Review by krol3 — COMMENTED** [review]
@gamba47 coloque comentarios como la palabra auto escalado separado,  y relacionado tb a la descripcion de Como ayuda, tengo comentarios tb que aun no fueron aplicados.

File: content/es/auto_scaling.md (hunk 2 above)
Comment (krol3) [reply content/es/auto_scaling.md]: @gamba47 revisa esta parte tb por favor

**Comment by gamba47** [comment]
Closing and recreating another pr based on the last meeting

**Pull request closed without merging**